cmd/notation/key.go

```diff
@@ -167,7 +179,8 @@ func addKey(command *cobra.Command, opts *keyAddOpts) error {
 	}
 	pluginName := opts.plugin
 	if pluginName != "" {
-		key, err = addExternalKey(command.Context(), opts, pluginName, name)
+		logger.Debug("Adding external key...")
```

Comment:
usually add more information/context before executing the command
```suggestion
		logger.Debug("Adding key with name %v and plugin name %v, key, pluginName")
```